data/json/items/armor/ballistic_armor.json

```diff
@@ -1160,7 +1160,14 @@
         "name": { "str": "press ballistic vest" },
         "description": "Colored a distinctive navy blue and with a flared collar extending up to the chin, this soft body armor was once used by news crews reporting from combat zones.  It provides good protection from shrapnel, and to a lesser degree, low-caliber bullets.  \"PRESS\" is emblazoned prominently on the front lapel and across the back, and it features a pocket for a notebook and pen.",
         "color": "blue"
-      }
+      },
+      {
+        "id": "fbi_armor_vest",
+        "name": { "str": "FBI ballistic vest" },
+        "//": "Based on the exact model; https://www.tyrtactical.com/shop/tyr-tactical-male-epic-federal-outer-carrier/",
+        "description": "A ranger green soft ballistic vest with a flared collar and MOLLE rigging used by federal law enforcement.  Soft armor plates sealed inside the fabric provide solid defense against shrapnel and small-caliber rounds. \"FBI\" is emblazoned across the front and back in large, gold lettering, and it includes a pocket for storing evidence.  Use it to attach or detach pouches.",
```

Comment:
```suggestion
        "description": "A ranger green soft ballistic vest with a flared collar and PALS rigging used by federal law enforcement agencies.  Soft armor plates inside the vest provide solid defense against shrapnel and small-caliber rounds. \"FBI\" is emblazoned across the front and back in large, gold lettering, and it includes a pocket for storing evidence.  Use it to attach or detach pouches.",
```

Reply:
The vest is specifically stated to have MOLLE rigging on the website